src/App.test.js

```diff
@@ -28,7 +28,7 @@ const mockFetch = () => {
     }));
 }
 
-describe('Fazer uma requisição para o endpoint `/planets` da API de Star Wars e preencher uma tabela com os dados retornados, com exceção dos da coluna `residents`', () => {
+describe('Faça uma requisição para o endpoint `/planets` da API de Star Wars e preencha uma tabela com os dados retornados, com exceção dos da coluna `residents`', () => {
```

Comment:
```suggestion
describe('Faça uma requisição para o endpoint `/planets` da API de Star Wars e construa uma tabela com os dados retornados, com exceção dos da coluna `residents`', () => {
```
O que acha? Não lembro se vocês tinham que fazer a tabela do zero.